### PR TITLE
Mslq 76

### PR DESCRIPTION
**Pull request by cacadam89**
main things to check: confirm wording is clear, and let me know if there is anything else that is needed

### CODE REVIEW

File: docs/source/software/mission_control.rst [hunk 1]
```diff
@@ -2,4 +2,17 @@
...
+
+For information regarding: 
+
+- Optitrack (our Motion Capture System)
+- Connecting to the Local Network
+- Using ROS
+- Our current Flightroom-specific computer setup 
+- and much, much, more
+
+see the `MSL FLightroom Documentation <https://stanfordflightroom.github.io/documentation>`_!
```
Comment (rhaksar) [inline docs/source/software/mission_control.rst]: Fix capitalization in "FLightroom"

**Comment by rhaksar** [comment]
I think the title "Mission Control" is too ambiguous, I think "Ground Station" or "Ground Control" might be better. @exoticDFT

**Comment by exoticDFT** [comment]
> I think the title "Mission Control" is too ambiguous, I think "Ground Station" or "Ground Control" might be better. @exoticDFT

Let's discuss this at today's meeting before finalizing this.

**Comment by rhaksar** [comment]
> > I think the title "Mission Control" is too ambiguous, I think "Ground Station" or "Ground Control" might be better. @exoticDFT
> 
> Let's discuss this at today's meeting before finalizing this.

We finalized it, yeah? Go ahead and merge the changes?

**Pull request merged**